daemon/daemon.go

```diff
@@ -38,13 +38,13 @@ func NewWebhookDaemonFromConfig(ctx context.Context, cfg *config.WebhookConfig)
 	d, err := NewWebhookDaemon(ctx, cfg.Daemon)
 
 	if err != nil {
-		return nil, fmt.Errorf("Failed to create new webhookd daemon, %w", err)
+		return nil, fmt.Errorf("failed to create new webhookd daemon, %w", err)
```

Comment:
Why have you changed the case for all the error messages? The convention for `go-webhookd` is title-cased error messages.

Reply:
Updated these messages as per guidance.  Changed originally due to lint errors.  I added a staticcheck.conf to remove these linting errors.